src/Translations/TranslationManager.php

```diff
@@ -102,7 +102,12 @@ protected function getPhpTranslations(string $folder)
     {
         $files = collect($this->filesystem->files($folder));
 
-        return $files->mapWithKeys(function ($file) {
+        return $files->reject(function ($file) {
+            return collect(config('poeditor-sync.excluded_files'))
+                    ->contains(function ($excluded_file) use ($file) {
+                        return $excluded_file.'.php' == $file->getFilename();
```

Comment:
would the array items of `config('poeditor-sync.excluded_files')` not have a file-extension already?

Reply:
I thought it would we cleaner in the config array without extension, like this:
```
'excluded_files' => [
    'auth',
    'validation'
]
```
Instead of this:
```
'excluded_files' => [
    'auth.php',
    'validation.php'
]
```

Reply:
bump (comment test)

Reply:
bump again

Reply:
bump